config/samples/udg2/udg_external_graphql_internal_rest.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: tyk.tyk.io/v1alpha1
+kind: ApiDefinition
+metadata:
+  name: rest-countries
+spec:
+  name: rest-countries
+  use_keyless: true
+  protocol: http
+  active: true
+  proxy:
+    target_url: https://restcountries.com/v2/alpha/
+    listen_path: /restCountries
+    strip_listen_path: true
+  version_data:
+    default_version: Default
+    not_versioned: true
+    versions:
+      Default:
+        name: Default
+
+
+
+---
+
+
+apiVersion: tyk.tyk.io/v1alpha1
+kind: ApiDefinition
+metadata:
+  name: udg
+spec:
+  name: Universal Data Graph v2b
+  use_keyless: true
+  protocol: http
+  active: true
+  proxy:
+    target_url: ""
+    listen_path: /udg
+    strip_listen_path: true
+  version_data:
+    default_version: Default
+    not_versioned: true
+    versions:
+      Default:
+        name: Default
+  graphql:
+    enabled: true
+    execution_mode: executionEngine
+    schema: |
+      type Country {
+        name: String
+        code: String
+        restCountry: RestCountry
+      }
+
+      type Query {
+        countries: [Country]
+      }
+
+      type RestCountry {
+        altSpellings: [String]
+        subregion: String
+        population: Int
+      }
+    version: "2"
+    last_schema_update: "2022-10-12T14:27:55.511+03:00"
```

Comment:
Should we create a UDG ApiDefinition with `last_schema_update`? I believe users do not need to provide it while creating their APIs.

Reply:
I left it here as an example so users know when the schema was last updated as trying to get a new schema from the UI will still be meaningless given that the operator will overwrite any modifications.